protocols/dcutr/src/protocol/inbound.rs

```diff
@@ -107,7 +110,7 @@ impl PendingConnect {
             self.substream
                 .next()
                 .await
-                .ok_or(super::codec::Error::Io(std::io::Error::new(
+                .ok_or(prost_codec::Error::Io(std::io::Error::new(
                     std::io::ErrorKind::UnexpectedEof,
                     "",
                 )))??;
```

Comment:
Maybe also add for dcutr a `{inbound, outbound}::UpgradeError::StreamClosed` like it was done for identify?

Reply:
Good idea. Done with the recent commit.